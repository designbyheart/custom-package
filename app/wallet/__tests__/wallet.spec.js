import React from 'react'
import renderer from 'react-test-renderer'
// import Wallet from '../wallet'

describe('<Wallet />', () => {
  xit('should render Wallet', () => {
    // const wrapper = renderer.create(<Wallet />).toJSON()
    // expect(wrapper).toMatchSnapshot()
  })
})
