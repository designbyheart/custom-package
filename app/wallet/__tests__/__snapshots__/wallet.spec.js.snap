// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Wallet /> should render Wallet 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#ffffff",
      "flex": 1,
    }
  }
  tertiary={true}
>
  <View
    screenProps={
      Object {
        "navigation": Object {
          "dispatch": [MockFunction],
          "getParam": [MockFunction],
          "goBack": [MockFunction],
          "isFocused": [MockFunction],
          "navigate": [MockFunction],
          "push": [MockFunction],
          "setParams": [MockFunction],
          "state": Object {
            "params": Object {},
          },
        },
      }
    }
  />
</View>
`;
